Makefile

```diff
@@ -26,7 +26,7 @@ clean:
 
 # Targets for testing
 test:
-	$(PYTHON_BIN)/nosetests -v --with-coverage --cover-package=$(PROJECT) --cover-inclusive --cover-erase tests
+	$(PYTHON_BIN)/py.test --cov=$(PROJECT) tests/
```

Comment:
I think that we should [#dropthedot](https://twitter.com/davehunt82/status/756080181085872129) given that `py.test` is deprecated.